File: 백준/Bronze/30033/30033_30033.js
/*
----------------------------------------------
Sub : [BOJ] Rust Study
Link: https://www.acmicpc.net/problem/30033
Tag : JS, 
Memo
----------------------------------------------
*/

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

// 데이터 처리
const N = parseInt(input[0]); // 일수 N
const A = input[1].split(' ').map(Number); // 계획한 페이지 수 A
const B = input[2].split(' ').map(Number); // 실제 공부한 페이지 수 B

let count = 0;
for (let i = 0; i < N; i++) {
  if (B[i] >= A[i]) {
    count++;
  }
}

// 결과 출력
console.log(count);

File: 백준/Bronze/30033/30033_30033.py
'''
----------------------------------------------
Sub : [BOJ] Rust Study
Link: https://www.acmicpc.net/problem/30033
Tag : Python
Memo
----------------------------------------------
'''

N = int(input())
A = list(map(int, input().split()))
B = list(map(int, input().split()))
count = 0

for i in range(N):
    if B[i] >= A[i]:
        count += 1

print(count)
